src/js/components/EmptyStateView.jsx

```diff
@@ -37,6 +51,6 @@ export class EmptyStateView extends Component {
 
 EmptyStateView.propTypes = {
     children: PropTypes.node,
-    iconName: PropTypes.oneOf(['branch','goat','shoes']),
+    iconName: PropTypes.string,
```

Comment:
because... ? this allows any material icon type, I don't think it's better to hardcode all this stuff further